angular-segmentio.js

```diff
@@ -7,6 +7,20 @@ angular.module('segmentio', ['ng'])
             //but do not attempt to create 'analytics' object.
             var tempQueue = [];
 
+
+            function flushTempQueue () {
+                // Send the queue of pending events
+                $window.analytics.push.apply($window.analytics, tempQueue)
+                tempQueue.length = 0
+            }
+
+            //  Analytics script is loaded callback.
+            var analyticsLoaded = function() {
+                if (tempQueue.length) {
```

Comment:
I would move the check to flushQueue :surfer: This way it can always be called safely.